test/fixtures/index.html

```diff
@@ -6,5 +6,12 @@
   </head>
   <body>
     <h1>Browser Polyfill Test Page</h1>
+
+    <!--
+         The following two DOM elements ensure that the polyfill doesn't detect the
+         globals defined for these DOM element ids as the Extensions API objects.
+    -->
+    <div id="browser"></div>
```

Comment:
Add another `<div id="browser"></div>` to demonstrate the issue that I mentioned above.